src/endless/EndlessUsbToolDlg.cpp

```diff
@@ -297,19 +297,21 @@ enum endless_action_type {
 #define FILE_GZ_EXTENSION	"gz"
 #define FILE_XZ_EXTENSION	"xz"
 
-#define RELEASE_JSON_URLPATH    _T("https://d1anzknqnc1kmb.cloudfront.net/")
-#define JSON_LIVE_FILE          "releases-eos-3.json"
-#define JSON_INSTALLER_FILE     "releases-eosinstaller-3.json"
-#define JSON_GZIP               "." FILE_GZ_EXTENSION
-#define JSON_PACKED(__file__)   __file__ JSON_GZIP
+#define RELEASE_JSON_URLPATH       _T("https://d1anzknqnc1kmb.cloudfront.net/")
+#define JSON_LIVE_FILE             "releases-eos-3.json"
+#define JSON_CODING_LIVE_FILE      "releases-coding-3.json"
+#define JSON_INSTALLER_FILE        "releases-eosinstaller-3.json"
+#define JSON_CODING_INSTALLER_FILE "releases-codinginstaller-3.json"
```

Comment:
I don't think you actually want to build `codinginstaller` images; we should always use the `eosinstaller` images.

Some background: once upon a time, you had to choose. A live USB was just a normal image written to a USB, and you couldn't install from it. An installer USB was an `eosinstaller` image with an extra exFAT partition containing image file(s) to install; but you couldn't use this as a live USB to try the image you're about to install.

Now, there's a "combined" USB which can do both. This is achieved (essentially) by loopback-mounting the uncompressed, pristine image from an exFAT partition, and including the `eos-installer` binary (note the hyphen!) inside all images. In this model, the separate `eosinstaller` image is not used.

Unless you [hold the `Control` key while clicking the “advanced options” link, or are running Windows XP](https://github.com/endlessm/rufus/blob/master/src/localization.c#L409), this app only gives you a way to create new-style combined USBs.

Given that `eosinstaller` images are only used in a “cheat mode”, I don't think you want to introduce a `codinginstaller` product.